docs/src/main/sphinx/connector/alter-mv-set-properties-unsupported.fragment

```diff
@@ -0,0 +1,6 @@
+ALTER MATERIALIZED VIEW SET PROPERTIES
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The connector does not support :ref:`ALTER MATERIALIZED VIEW SET PROPERTIES
+<alter-materialized-view-set-properties>` statements. Other
+:doc:`/sql/alter-materialized-view` statements are supported.
```

Comment:
Remove the `Other` part since once we have multiple such fragments (which we'll do once more syntax is added) are added this statement will become false. (And it would not be caught in review since it's a fragment).

Reply:
Done